samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/App.kt

```diff
@@ -104,6 +107,7 @@ class App {
                             onClickIsoMdocProximitySharing = { navController.navigate(IsoMdocProximitySharingDestination.route) },
                             onClickIsoMdocProximityReading = { navController.navigate(IsoMdocProximityReadingDestination.route) },
                             onClickMdocTransportMultiDeviceTesting = { navController.navigate(IsoMdocMultiDeviceTestingDestination.route) },
+                            onClickCerificatesViewer = { navController.navigate(CertificatesViewerExamplesDestination.route) },
```

Comment:
I think `onClickCerificatesViewer` should probably be called `onClickCertificatesViewerExamples`.